test/tests/neml_benchmark/linear_strain_hardening/gold/elas_plas_LSH_out.csv

```diff
@@ -0,0 +1,22 @@
+time,_dt,disp_x1,disp_x4,disp_y1,disp_y4,eff_plastic_strain,el_strain_xx,el_strain_yy,el_strain_zz,pl_strain_xx,pl_strain_yy,pl_strain_zz,pressure,stress_xx,stress_xy,stress_yy,stress_zz,tot_strain_xx,tot_strain_yy,tot_strain_zz,vonmises
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+0.1,0.1,0,0,0,2.08e-06,0,-6.9333261210831e-07,2.0799978368803e-06,0,0,0,0,231110.87079534,5.5538490414619e-05,0,554666.08985327,138666.5224772,-6.9333261210831e-07,2.0799978368803e-06,0,499969.25688941
```

Comment:
It looks like these results differ from the native MOOSE ones by many orders of magnitude. We should have the two models use the same units -- I can't think of a reason why not to. These are just made-up materials used for testing the models.